distribution/lib/Standard/Table/0.0.0-dev/src/Data/Type/Value_Type.enso

```diff
@@ -140,13 +141,15 @@ type Value_Type
        In-Memory and SQLite tables support this.
     Mixed
 
-    ## Checks if the `Value_Type` represents a boolean type.
+    ## GROUP Standard.Base.Metadata
+       Checks if the `Value_Type` represents a boolean type.
```

Comment:
I'm wondering if these need a group. They are really mostly for the internal use to check if the type is fitting. I'd consider removing the GROUP and adding `PRIVATE ADVANCED` pair.

Reply:
But also fine with keeping as-is. I imagine these methods _can_ be useful to end-users and they are not magic that needs to be hidden.